tests/tools/fio/fio.go

```diff
@@ -62,8 +64,23 @@ type Runner struct {
 	FioWriteBaseDir string
 	Global          Config
 	Debug           bool
+
+	PathLock pathlock.Locker
 }
 
+// InactivePathLocker satisfies the pathlock.Locker interface but is a no-op.
+type InactivePathLocker struct{}
+
+var _ pathlock.Locker = (*InactivePathLocker)(nil)
+
+// Lock implements the pathlock.Locker interface.
+func (l *InactivePathLocker) Lock(lockPath string) (pathlock.Unlocker, error) {
+	return l, nil
+}
+
+// Unlock satisfies the pathlock.Unlocker interface.
+func (l *InactivePathLocker) Unlock() {}
```

Comment:
nit: `NullPathLocker` follows the convention for other no-op structs in the repo

```suggestion
// NullPathLocker satisfies the pathlock.Locker interface but is a no-op.
type NullPathLocker struct{}

var _ pathlock.Locker = (*NullPathLocker)(nil)

// Lock implements the pathlock.Locker interface.
func (l *NullPathLocker) Lock(lockPath string) (pathlock.Unlocker, error) {
	return l, nil
}

// Unlock satisfies the pathlock.Unlocker interface.
func (l *NullPathLocker) Unlock() {}
```